gensim/test/test_sklearn_integration.py

```diff
@@ -67,5 +84,21 @@ def testCSRMatrixConversion(self):
             self.assertTrue(isinstance(v, six.string_types))
             self.assertTrue(isinstance(k, int))
 
+    def testPipline(self):
+        model = SklearnWrapperLdaModel(num_topics=2, passes=10, minimum_probability=0, random_state=numpy.random.seed(0))
+        vec = CountVectorizer(min_df=10, stop_words='english')
+        rand = numpy.random.mtrand.RandomState(1) # set seed for getting same result
+        cats = ['rec.sport.baseball', 'sci.crypt']
+        data = fetch_20newsgroups(subset='train',
```

Comment:
there are smaller datasets in test_data folder. downloading a lot of data makes tests run too long